benchmarks/scripts/expval_benchmark.py

```diff
@@ -78,26 +80,44 @@ def compile_for_simulation(
         case _:
             raise ValueError(f"Unknown compiler alias: {compiler_alias}")
 
-
-def simulate_density_matrix(circuit: qiskit.QuantumCircuit) -> np.ndarray:
+def depolarizing_error_model(one_q_err, two_q_err):
     depolarizing_noise = NoiseModel()
-    error = depolarizing_error(0.01, 1)
-    two_qubit_error = depolarizing_error(0.02, 2)
+    error = depolarizing_error(one_q_err, 1)
+    two_qubit_error = depolarizing_error(two_q_err, 2)
     # TODO: errors should only be added to the gateset that we are compiling to
     # but there is a bug with cirq currently compiling to U3 and CZ
-    depolarizing_noise.add_all_qubit_quantum_error(
-        error, ["u1", "u2", "u3", "rx", "ry", "rz", "h"]
-    )
-    depolarizing_noise.add_all_qubit_quantum_error(
-        two_qubit_error, ["cx", "cz"]
-    )
+    depolarizing_noise.add_all_qubit_quantum_error(error, ["u1", "u2", "u3", "rx", "ry", "rz", "h"])
+    depolarizing_noise.add_all_qubit_quantum_error(two_qubit_error, ["cx", "cz"])
+    return depolarizing_noise
 
-    simulator = AerSimulator(
-        method="density_matrix", noise_model=depolarizing_noise
-    )
+
+def simulate_density_matrix(circuit: qiskit.QuantumCircuit) -> np.ndarray:
+    simulator = AerSimulator(method="density_matrix", noise_model=depolarizing_error_model())
     return simulator.run(circuit).result().data()["density_matrix"]
 
 
+def get_heavy_bitstrings(circuit: qiskit.QuantumCircuit) -> Set[str]:
+    simulator = AerSimulator(method="statevector")
+    result = simulator.run(circuit).result()
+    probs = list(result.get_counts().items())
+    median = np.median([p for (_, p) in probs])
```

Comment:
I haven't run this code explicitly, but when looking at qiskit's results in another context, it looked like the values in the `get_counts` returned dict are the number of times that bitstring occurred, versus a normalized probability.

Reply:
It's not being treated as a probability- so a better name would be `counts`. I used the [qv_batch_ibmq.ipynb](https://github.com/unitaryfund/mitiq-internal/blob/master/mitiq-metrics/volume/qv_batch_ibmq.ipynb) initially so some of the (admittedly informal) terminology made its way in.